autogen/json2class.cpp

```diff
@@ -259,9 +259,8 @@ void check_class(
    assert(value.contains("attributes"));
    assert(value.contains("childNodes"));
    assert(value.contains("description"));
-   assert(value.contains("bodyText"));
```

Comment:
OK, I'll put it back in. I'd had the impression from our conversation the other day that perhaps we should allow it to be optional. No big deal.

Reply:
Since these are asserts, it'll only be an issue when running in debug. In the future, we should refactor this as a separate tool to check the json file without generating code from it - we should do that once the prototype is ready.